decidim-participatory_processes/app/queries/decidim/participatory_processes/stats_participants_count.rb

```diff
@@ -58,6 +59,26 @@ def debates_query
           .uniq
       end
 
+      def meetings_query
+        meetings = Decidim::Meetings::Meeting
+                   .where(component: space_components)
+                   .not_hidden
+                   .pluck(:id)
+
+        registrations = Decidim::Meetings::Registration
+                        .where(decidim_meeting_id: meetings)
+                        .pluck(:decidim_user_id)
+                        .uniq
+        organizers = Decidim::Meetings::Meeting
+                     .where(
+                       decidim_author_type: Decidim::UserBaseEntity.name,
+                       id: meetings
+                     )
+                     .pluck(:decidim_author_id)
+                     .uniq
```

Comment:
💬 There are some performance improvements that can be done here:
* You don't need to retrieve all the meeting ids from the database and then send them again in the following queries. You can store the query on a variable, and then use it as a base for another query or embed it with a `where`. In many cases, Rails will know how to build the right SQL sentence.
* When retrieving unique values from the database it's better to perform the filter in the database than in the app, to avoid retrieving duplicated values that you will discard.

To sum up, this code should be equivalent to yours:

```suggestion
        meetings = Decidim::Meetings::Meeting
                   .where(component: space_components)
                   .not_hidden

        registrations = Decidim::Meetings::Registration
                        .where(decidim_meeting: meetings)
                        .distinct
                        .pluck(:decidim_user_id)
        				
        organizers = meetings
                    .where(decidim_author_type: Decidim::UserBaseEntity.name)
                    .distinct
                    .pluck(:decidim_author_id)
```